feature-libs/organization/administration/components/unit/details/unit-details.component.html

```diff
@@ -1,12 +1,19 @@
 <cx-org-card *ngIf="model$ | async as model" i18nRoot="orgUnit.details">
-  <a
+  <div
     actions
-    class="link edit"
-    [class.disabled]="!model.active || (isInEditMode$ | async)"
-    [routerLink]="{ cxRoute: 'orgUnitEdit', params: model } | cxUrl"
+    *ngIf="model.active"
+    [cxFocus]="{ autofocus: true }"
+    [cxRefreshFocusOn]="model.uid"
```

Comment:
The cxFocus comes with a configuration. I'd consider extending the `AutoFocusConfig` with a `refreshFocus` property.

Reply:
worth trying if that gives the right result though.